manifests/params.pp

```diff
@@ -6,6 +6,7 @@
 
   case $facts['os']['family'] {
     'Archlinux': {
+      $manage_python    = false
```

Comment:
do we need python only on debian like systems?

Reply:
I think we can require python on all platforms. It's ```ensure_packages``` so there's less chance for conflicts and otherwise we provide the option to disable it.

Reply:
We can if you think that makes sense. My thought was to take a conservative approach, since a) I don't know for sure which systems don't already have it, and b) I don't know if the python 2 package name is consistent across all those platforms, and we don't do acceptance tests on all of them. Once this is enabled, we could switch the param for other platforms. Thoughts?